### PR TITLE
proxr is interlock channel aware

### PR DESCRIPTION
**Pull request by 4Lowe**
First pass at channel support. Works well with my 2 relay controller I currently have for testing.

### CODE REVIEW

File: NEMO/interlocks.py [hunk 1]
```diff
@@ -227,29 +222,35 @@ def _send_bytes(self, relay_socket, proxrcmd):
...
+			# proxr relay controllers have a max of 8 relays (channels) per bank
+			raise InterlockError(interlock=interlock, msg="Invalid channel: " + str(interlock.channel))
 		state = Interlock_model.State.UNKNOWN
 		try:
 			with socket.create_connection((interlock.card.server, interlock.card.port), 10) as relay_socket:
 				if command_type == Interlock_model.State.LOCKED:
-					self._send_bytes(relay_socket, self.PXR_CMD_B1_OFF)
-					state = self._get_state(relay_socket)
+					# turn the interlock channel off
+					self._send_bytes(relay_socket, (254, 99 + interlock.channel, 1))
```
Comment (mrampant-nist) [inline NEMO/interlocks.py:248]: I didn't really look at the documentation to confirm, but do you know if those commands (99 + channel etc.) are the same for all web-based ProXr interlocks?

**Comment by 4Lowe** [comment]
I like the idea of using the interlock card number for the bank. I think this should be tabled for the future since I can't actually test right now. It should be easy enough to add.

Regarding the channel checking... funny, I had just changed the code to "in range(1, 9)". Let me know when you have clean_interlock ready.

As far as I know the ProXR commands are universal. What I know comes mostly from the [quick start guide](https://ncd.io/proxr-quick-start-guide/). Different controllers support different numbers of banks but I'm not sure of all the specifics.

You should also know about API mode which gives more reliable functioning. However, not all of our controllers support API mode. And, most of our controllers don't respond to the API mode query that is supposed to work. So fully supporting these relay controllers would be a tricky endeavor.

I'd be happy to write doc for the wiki that elaborates on what is supported and the model/revisions tested.

**Comment by mrampant-nist** [comment]
clean_interlock function is ready. you can look at the example for the stanford implementation.
if you could add some documentation [here](https://github.com/usnistgov/NEMO/wiki/Interlock-Setup) that'd be great. you might want to put the current WebRelay info in it's own chapter and add ProXr

**Comment by 4Lowe** [comment]
The channel validation has been moved to the interlock form.

**Comment by mrampant-nist** [comment]
perfect. thank you!

**Comment by mrampant-nist** [comment]
I added a placeholder on the wiki page. feel free to add the documentation whenever you have the time. no rush!

**Pull request merged**